js/webflow.js

```diff
@@ -20148,15 +20148,25 @@ Webflow.define('tabs', module.exports = function ($) {
   return api;
 });
 
-function reloadTheme(){
+function reloadTheme(checkPreference){
   var themeSwitcher = document.getElementById("theme-switcher");
-  var themeSwitcherTrueText = themeSwitcher.innerHTML.slice(0, themeSwitcher.innerHTML.indexOf(">") + 1); 
+  var themeSwitcherTrueText = themeSwitcher.innerHTML.slice(0, themeSwitcher.innerHTML.indexOf(">") + 1);
+
+  	if (checkPreference) {
```

Comment:
```suggestion
  	if (checkPreference && !localStorage.getItem('darkMode')) {
```
I think that if the user has changed the mode, they should be able to keep it

Reply:
Thanks for mentioning it, I didn't think of that.

Reply:
Nevermind, if this is added, it seems to break the logic entirely.

Reply:
It's for the people who don't have the darkMode field in their localstorage (their first time going on the site)
 It makes sense because if they choose to change the mode they should be able to keep it

Reply:
And it's that the mode set to their default when **first** visiting the site and not everytime they reload

Reply:
Yes, but after committing your code the pr seems to not work at all, and it always defaults to Light Mode.

Reply:
After I try to find a fix ill reopen the pr.

Reply:
I am currently working on a fix, should be done by tomorrow.